core/src/banking_stage/transaction_scheduler/scheduler_common.rs

```diff
@@ -245,7 +249,11 @@ impl<Tx: TransactionWithMeta> SchedulingCommon<Tx> {
 
                 debug_assert!(
                     retryable_iter.peek().is_none(),
-                    "retryable indexes were not in order: {retryable_indexes:?}"
+                    "retryable indexes were not in order: {:?}",
+                    retryable_indexes
```

Comment:
probably won't hurt to print entire `RetryableIndex`, more info for debugging

Reply:
it's the indexes not being in order that causes issue - no need to implement debug for the whole struct this way.